.github/workflows/scheduler-resources-test.yml

```diff
@@ -0,0 +1,20 @@
+name: test optimus resources
```

Comment:
why not integrate in make test-ci target, and we don't need another workflow for running unit tests.

Reply:
sure

Reply:
sure